test/common/router/route_fuzz.proto

```diff
@@ -5,6 +5,8 @@ package test.common.router;
 import "envoy/api/v2/rds.proto";
 import "test/fuzz/common.proto";
 
+import "validate/validate.proto";
```

Comment:
not used.

Reply:
Oh interesting, I figured this would also need to be built with validate since the messages used are. Seems like no!